be/fmt.h

```diff
@@ -262,6 +265,11 @@ struct m0_be_fmt_log_record_header {
 	m0_bindex_t                            lrh_prev_pos;
 	m0_bindex_t                            lrh_prev_size;
 	uint64_t                               lrh_io_nr_max;
+	/* lrh_serial_num is populated with flh_serial_num of log header.
+	 * These two fields are compared at the time of recovery
+	 * to check the validity of record.
+	 */
+	uint64_t			       lrh_serial_num;
```

Comment:
```
/**
 * This is a Doxygen comment.
 */
```
is t he correct format.

Please add explanations, what are the correct comparison result?
In what cases, we say a record is valid?